c++/src/kj/table.c++

```diff
@@ -288,27 +288,27 @@ void BTreeImpl::growTree(uint minCapacity) {
   // aligned_alloc() function. Unfortunately, many platforms don't implement it. Luckily, there
   // are usually alternatives.
 
-#if __APPLE__
```

Comment:
I'd prefer to add `__ANDROID__` (or perhaps better `__BIONIC__`) here, rather than make posix_memalign() the default. alligned_alloc() is part of the C11 standard, while posix_memalign() is POSIX-specific, so aligned_alloc() is the better default case. If there's a universal way to detect C11, that works too, but _ISOC11_SOURCE seems to be glibc-specific.